DigitalLearningSolutions.Web/ViewModels/MyAccount/UpdateNotificationPreferencesViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models;
+
+    public class UpdateNotificationPreferencesViewModel // TODO 

```

Comment:
TODO?

Reply:
Not sure what I was meaning to type there at the time but I've got rid of it now!